### ISSUE TITLE
s390x binaries need to be recompiled

### ISSUE THREAD

**Issue by jschintag** [mikefarah/yq#502]
The issue mentioned in #394 is now fixed in golang and backported to versions 1.13 and 1.14.
I tested compiling yq localy on s390x with go version 1.13.14. The resulting binary showed no problem with parsing yaml and passed `make local test`. I tested it with version `3.3.2` and version `3.1.0`.

Could i trouble you to rebuild the s390x binaries and release them?
I specifically currently need the yq `3.1.0` as it is used in the CI for kata-containers.

**Comment by mikefarah** [comment]
I have recompiled using the latest go lang compiler (1.15) here: Fixed https://github.com/mikefarah/yq/releases/tag/3.2.3

Please let me know if that works

**Comment by jschintag** [comment]
Yes, Version 3.2.3 works. But it also seems that the version inside the binary is wrong:
```
root@m8360003:~# ./yq_linux_s390x --version
yq version 3.3.3
root@m8360003:~# ./yq_linux_s390x r yq/examples/data1.yaml a
simple
root@m8360003:~#
```
It should be `3.2.3` right?

**Comment by mikefarah** [comment]
Yep got the versioning all confused sorry - I've re-released version 3.3.4.
